extension.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extension format="1.0">
     <extension_code>sv-display-mgmt</extension_code>
-    <version>1.1.1</version>
+    <company>Super-Vision</company>
+    <author><![CDATA[Hipska]]></author>
     <label><![CDATA[Display management]]></label>
     <description><![CDATA[Document displays of various types]]></description>
+    <version>1.2.0</version>
+    <release_date>2024-11-04</release_date>
+    <version_description><![CDATA[]]></version_description>
+    <itop_version_min>3.2.0</itop_version_min>
+    <status>stable</status>
```

Comment:
Well,  this will be usefull the day where the extension will be publish on iTop Hub. This may not happen tomorrow but at least you'd be ready.
The publication day is the one you, as the extension owener, select to say, from now on, this version is "released".